sdk-workflows/src/main/java/io/dapr/workflows/client/DaprWorkflowClient.java

```diff
@@ -0,0 +1,121 @@
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.workflows.client;
+
+import com.microsoft.durabletask.DurableTaskClient;
+import com.microsoft.durabletask.DurableTaskGrpcClientBuilder;
+import io.dapr.config.Properties;
+import io.dapr.utils.Version;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+
+import javax.annotation.Nullable;
+
+public class DaprWorkflowClient implements AutoCloseable {
+
+  private DurableTaskClient innerClient;
+  private ManagedChannel grpcChannel;
+
+  /**
+   * Public constructor for DaprWorkflowClient. This layer constructs the GRPC Channel.
+   */
+  public DaprWorkflowClient() {
+    this(createGrpcChannel());
+  }
+
+  /**
+   * Private Constructor that passes a created DurableTaskClient and the new GRPC channel.
+   *
+   * @param grpcChannel ManagedChannel for GRPC channel.
+   */
+  private DaprWorkflowClient(ManagedChannel grpcChannel) {
+    this(createDurableTaskClient(grpcChannel), grpcChannel);
+  }
+
+  /**
+   * Private Constructor for DaprWorkflowClient.
+   *
+   * @param innerClient DurableTaskGrpcClient with GRPC Channel set up.
+   * @param grpcChannel ManagedChannel for instance variable setting.
+   *
+   */
+  private DaprWorkflowClient(DurableTaskClient innerClient, ManagedChannel grpcChannel) {
+    this.innerClient = innerClient;
+    this.grpcChannel = grpcChannel;
+  }
+
+  /**
+   * Static method to create the DurableTaskClient.
+   *
+   * @param grpcChannel ManagedChannel for GRPC.
+   * @return a new instance of a DurableTaskClient with a GRPC channel.
+   */
+  private static DurableTaskClient createDurableTaskClient(ManagedChannel grpcChannel) {
+    return new DurableTaskGrpcClientBuilder()
+        .grpcChannel(grpcChannel)
+        .build();
+  }
+
+  /**
+   * Static method to create the GRPC Channel for the DurableTaskClient.
+   *
+   * @return a Managed GRPC channel.
+   * @throws IllegalStateException if the GRPC port is invalid.
+   */
+  private static ManagedChannel createGrpcChannel() throws IllegalStateException {
+    int port = Properties.GRPC_PORT.get();
+    if (port <= 0) {
+      throw new IllegalStateException("Invalid port.");
+    }
+
+    ManagedChannel channel =  ManagedChannelBuilder.forAddress(Properties.SIDECAR_IP.get(), port)
+        .usePlaintext()
+        .userAgent(Version.getSdkVersion())
+        .build();
+    return channel;
+  }
+
+  /**
+   * Schedules a new workflow using DurableTask client.
+   *
+   * @param workflowName name of workflow to start.
+   * @return String for new Workflow instance id.
+   */
+  public String scheduleNewWorkflow(String workflowName) {
```

Comment:
Consider adding an overload that allows the user to specify an instance ID. This is common, especially for workflows that correspond to external entities (document workflows, etc.) and helps users who need their workflows to be singletons.